checker/check_result.go

```diff
@@ -17,12 +17,12 @@ package checker
 const MaxResultConfidence = 10
 
 type CheckResult struct {
+	Error       error `json:"-"`
 	Name        string
-	Pass        bool
+	Details     []string `json:",omitempty"`
```

Comment:
Let's remove the "omitempty" option here. That should allow us to submit the PR without any breaking change?

Reply:
Yep, that will make the PR not have any breaking changes. I have removed that option.